### DISCUSSION TITLE
Unicode Not Rendering in Cell Output in Pre-Release v2022.10.1102941155

### DISCUSSION THREAD

**Discussion by AJTonyKane** [microsoft/vscode-jupyter#11712] [General] [answered]
I have found that unicode output has changed / in not working as before on the most recent Visual Studio Code Jupyter Pre-Release.  In generating a progression bar, I use left+right arrow (↹) icon to signify activity ('\u21B9') then print a backup character so that it is subsequently erased, progressing with a series of unicode encircled numbers, like an encircled 1 (➀):  '\b\u2780' .  The new pre-release version doesn't seem to handle properly and I'm not sure what setting may change 

Jupyter Release v2022.9.1202862440:
DBMS Commits:➀➁➂➃➄➅➆➇➈✦➀➁➂➃➄➅➆➇➈✦➀➁➂➃➄ ✨ Complete!

Jupyter Pre-Release v2022.10.1102941155:
DBMS Commits:↹➀↹➁↹➂↹➃↹➄↹➅↹➆↹➇↹➈↹✦↹➀↹➁↹➂↹➃↹➄↹➅↹➆↹➇↹➈↹✦↹➀↹➁↹➂↹➃↹➄↹ ✨ Complete!

**Comment by DonJayamanne** [comment, marked as answer]
Thanks for bringing this to our attention, what version of VS Code are you using?
Also, please could you provide some sample code that we could use to replicate this issue at our end.

**Reply by AJTonyKane** [reply]
I am using VS code version 1.72.2.  The attached .txt file contains sample python code (.ipynb & .py extensions are not supported):  

Many thanks for your attention on this!

Regards and best wishes,
Tony

![image](https://user-images.githubusercontent.com/19287204/197020566-82629bb6-109a-4b41-ace3-8dc9c0c1884d.png)

[dbms batch commits prog bar demo.txt](https://github.com/microsoft/vscode-jupyter/files/9832804/dbms.batch.commits.prog.bar.demo.txt)

**Reply by DonJayamanne** [reply]
Aah, please use the latest insiders version of VS Code and I believe that will fix this.
This months pre-relase version of the Jupyter extesnions have a change that depends on a corresponding fix in VS Code, and the fix is currently only avaialble in the insider version of VS Code.

I've tested you sample code (with the insider version of VS Code) and it works as expected, see below.
![Screen Shot 2022-10-21 at 06 15 45](https://user-images.githubusercontent.com/1948812/197037927-cd4f7ffa-c56b-4f64-978d-63a363a1852a.png)

**Reply by AJTonyKane** [reply]
Will do. Tagging as having been answered. Thanks so much Don!
